### PR TITLE
feat(elasticbeanstalk): add new check `elasticbeanstalk_cloudwatch_enabled`

### PR DESCRIPTION
**Pull request by MarioRgzLpz**
### Context

Add a new check that verifies whether an Elastic Beanstalk environment is configured to send logs to CloudWatch Logs. The check fails if the environment is not configured to send logs to CloudWatch Logs.

CloudWatch helps you collect and monitor various metrics for your applications and infrastructure resources. You can also use CloudWatch to configure alarm actions based on specific metrics. We recommend integrating Elastic Beanstalk with CloudWatch to get increased visibility into your Elastic Beanstalk environment. Elastic Beanstalk logs include the eb-activity.log, access logs from the environment nginx or Apache proxy server, and logs that are specific to an environment.

### Description

Add new check `elasticbeanstalk_cloudwatch_enabled` with respective unit tests and metadata.

### Checklist

- Are there new checks included in this PR? Yes 
    - If so, do we need to update permissions for the provider? No
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "elasticbeanstalk_cloudwatch_enabled",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.metadata.json]:
```suggestion
  "CheckID": "elasticbeanstalk_environment_cloudwatch_logging_enabled",
```

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.metadata.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "CheckTitle": "Elastic Beanstalk should stream logs to CloudWatch",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.metadata.json]:
```suggestion
  "CheckTitle": "Elastic Beanstalk environment should stream logs to CloudWatch",
```

File: prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            report.resource_id = environment.name
+            report.resource_arn = environment.arn
+            report.resource_tags = environment.tags
+            report.status = "PASS"
+            report.status_extended = f"Elastic Beanstalk environment {environment.name} is sending logs to CloudWatch Logs."
+
+            if (
+                environment.cloudwatch_stream_logs is None
+                or environment.cloudwatch_stream_logs != "true"
+            ):
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/elasticbeanstalk/elasticbeanstalk_cloudwatch_enabled/elasticbeanstalk_cloudwatch_enabled.py]:
```suggestion
            if environment.cloudwatch_stream_logs != "true":
```

**Comment by MarioRgzLpz** [comment]
@sergargar Requested changes done [0b8123b](https://github.com/prowler-cloud/prowler/pull/5335/commits/0b8123b68719ca66c3bb3922e3fd7668d31ceff0)

**Review by MrCloudSec — APPROVED** [review]

**Pull request merged**